File: MusicWeb.Client/src/models/AlbumReview.js
export default class AlbumReview {
  id = "";
  title = "";
  content = "";
  postDate = "";
  albumId = "";
  userId = "";
  name = "";
  artist = "";
  userName = "";
  artist = "";
  rating = "";
  constructor($data) {
    if ($data) {
      this.id = $data.id;
      this.title = $data.title;
      this.content = $data.content;
      this.postDate = $data.postDate;
      this.albumId = $data.albumId;
      this.userId = $data.userId;
      this.name = $data.name;
      this.artist = $data.artist;
      this.userName = $data.userName;
      this.artist = $data.artist;
      this.rating = $data.rating;
    }
  }
}

File: MusicWeb.Server/MusicWeb.Models/Entities/Albums/AlbumReview.cs
using MusicWeb.Models.Entities.Base;
using MusicWeb.Models.Entities.Ratings;
using MusicWeb.Models.Identity;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MusicWeb.Models.Entities
{
    public class AlbumReview : BaseEntity
    {
        public string Title { get; set; }
        public string Content { get; set; }
        public DateTime PostDate { get; set; }
        public int AlbumId { get; set; }
        public string UserId { get; set; }

        public virtual Album Album { get; set; }
        public virtual ApplicationUser User { get; set; }

    }
}
